src/HealthChecks.AzureServiceBus/AzureEventHubHealthCheck.cs

```diff
@@ -13,8 +13,11 @@ public class AzureEventHubHealthCheck
         private readonly string _eventHubName;
         public AzureEventHubHealthCheck(string connectionString, string eventHubName)
         {
-            _connectionString = connectionString ?? throw new ArgumentNullException(nameof(connectionString));
-            _eventHubName = eventHubName ?? throw new ArgumentNullException(nameof(eventHubName));
```

Comment:
probably mix features on different package is not a good idea for tracking project. Can you split in two different pull requests

Reply:
New tests can't pass without this changes to the package. :/